packages/ilios-common/addon/components/timed-release-schedule.js

```diff
@@ -17,6 +17,6 @@ export default class TimedReleaseSchedule extends Component {
     if (!this.args.startDate) {
       return false;
     }
-    return DateTime.fromJSDate(this.args.startDate) > DateTime.now();
+    return DateTime.fromJSDate(new Date(this.args.startDate)) > DateTime.now();
```

Comment:
Probably don't need the interim `Date`, if this is a string coming in and it is correctly formatted you can use [fromISO](https://moment.github.io/luxon/#/parsing?id=iso-8601) directly.

```suggestion
    return DateTime.fromISO(this.args.startDate) > DateTime.now();
```

Reply:
I tried that one, but tests failed. I will change it and fix the tests instead.

Reply:
Ah right, because tests were assuming at `Date` object. Yes, update the tests to match reality and then I guess we'll go full typescript. I see no reasonable alternative to check values like this and get a good error 😖

Reply:
Offline conversation, we're going to move forward with this fix and circle back for cleanup.